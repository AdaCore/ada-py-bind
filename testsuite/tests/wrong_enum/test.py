import gen

p = gen.Shape()
print p.kind
p.kind = "circle"
print p.kind

try:
    p.kind ="lol"
except Exception as e:
    print e.message

print p.kind
