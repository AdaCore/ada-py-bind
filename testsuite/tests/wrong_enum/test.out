/home/amiard/libadalang/install/libadalang_env/bin/python returned status code 1
demo.ads:71:04: instantiation error at py_bind-types.ads:100
demo.ads:71:04: T has to be an enumeration type
gprbuild: *** compilation phase failed

